connector_test.go

```diff
@@ -210,10 +204,6 @@ func TestContainerName(t *testing.T) {
 		assert.Equals(t, time.Now().Before(end), true)
 		time.Sleep(1 * time.Second)
 	}
-
-	wg.Wait()
```

Comment:
Why is the waitgroup removed?

Reply:
I removed it because I thought it wasn't needed...was I mistaken?

Reply:
I see how the new design doesn't require it. Hopefully the change in behavior doesn't have any unintended side effects.

Reply:
To be honest, I don't see how the old design required it, either.  (I'd be happy if you could enlighten me....)

I don't think that removing the wait-groups will produce any visible change in behavior, let alone (execution-time) side effects.  I think it will just simplify the code.  (Removing the timing-sleeps will hopefully have the "side-effect" of removing the flakiness from the tests.)